### DISCUSSION TITLE
1.5.0.3

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#470] [Обновления]
## Список изменений
- Стандартный прокси-воркер изменен на `https://vot-worker.onrender.com`

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0.2...1.5.0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0.3'>1.5.0.3</a>.</em>

**Comment by NullVerdict** [comment]
> ## Список изменений
> 
>     * Стандартный прокси-воркер изменен на `https://vot-worker.onrender.com`
> 
> 
> **Full Changelog**: [1.5.0.2...1.5.0.3](https://github.com/ilyhalight/voice-over-translation/compare/1.5.0.2...1.5.0.3)
> _This discussion was created from the release [1.5.0.3](https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0.3)._

А что с Deno стало?

**Reply by ilyhalight** [reply]
> > ## Список изменений
> > ```
> > * Стандартный прокси-воркер изменен на `https://vot-worker.onrender.com`
> > ```

> > **Full Changelog**: [1.5.0.2...1.5.0.3](https://github.com/ilyhalight/voice-over-translation/compare/1.5.0.2...1.5.0.3)
> > _This discussion was created from the release [1.5.0.3](https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0.3)._
> 
> А что с Deno стало?

Почти лимит набрался. Не думал, что всего за неделю, миллион запросов наберется

![изображение](https://github.com/ilyhalight/voice-over-translation/assets/62353659/439ccd87-22e9-4481-b7da-d2e31e45c855)

**Comment by NullVerdict** [comment]
> > > ## Список изменений
> > > ```
> > > * Стандартный прокси-воркер изменен на `https://vot-worker.onrender.com`
> > > ```
> 
> > > **Full Changelog**: [1.5.0.2...1.5.0.3](https://github.com/ilyhalight/voice-over-translation/compare/1.5.0.2...1.5.0.3)
> > > _This discussion was created from the release [1.5.0.3](https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0.3)._
> > 
> > 
> > А что с Deno стало?
> 
> Почти лимит набрался. Не думал, что всего за неделю, миллион запросов наберется
> 
> ![изображение](https://private-user-images.githubusercontent.com/62353659/293393892-439ccd87-22e9-4481-b7da-d2e31e45c855.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDM4NzM1MTYsIm5iZiI6MTcwMzg3MzIxNiwicGF0aCI6Ii82MjM1MzY1OS8yOTMzOTM4OTItNDM5Y2NkODctMjJlOS00NDgxLWI3ZGEtZDJlMzFlNDVjODU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyMzEyMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI5VDE4MDY1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwODkzNTE2NDIyY2NhYjJhOWQwYzUwNjI1N2M3NDU1Y2VhOGRlZDI5OGIzMDViNTM0MGFmNmIxZjlhZDQxYjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rYqlHoWtXR2YQwxD27hZDT9q_zjUf5xsea2Wv-JLLn8)

Тем временем сервис определение языка опять отдыхает... Мб мне стоит переписать свой сервис под апи яндекса? 
Просто у меня вычисления происходят локально, чтобы вдруг Яндекс в лимит не кинул...
UPD: Ожил
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/c681325b-6c70-406e-8015-fbb72dcb400d)

**Reply by ilyhalight** [reply]
пофиксил

**Comment by NullVerdict** [comment]
> пофиксил

А почему он лёг?

**Reply by ilyhalight** [reply]
Скорее всего хостер у которого держу делал что-то с сетью и из-за этого сервер лаганул. Обычная перезагрузка все решила

**Reply by ilyhalight** [reply]
а нет, не пофиксилось. Все-таки это какие-то проблемы с сетью, ибо сейчас даже ссш умер
